testData/psi/KeywordLikeIdentifiersAsIdentifiers.smk

```diff
@@ -8,4 +8,5 @@ rule foo: run:
     checkpoint = 1
     configfile = 2
     report = 3
-    wildcard_constraints = 4
\ No newline at end of file
+    wildcard_constraints = 4
+    print(rule)
```

Comment:
Could you add also a test when `print(rule)` is not only in `run` section? is `print(rule)` is example when parsing is broken due to statement break inserted by lexer? If not let's add such test. If yes, please also add testcase to lexer test because it is also a lexer-related issue.